mkdocs.yml

```diff
@@ -313,6 +15,10 @@ theme:
     - tabs
     - instant
     - search.highlight
+    - navigation.sections
+    - navigation.path
+    - navigation.indexes
+    - toc.follow
```

Comment:
@felker I think this fixes the TOC in right sidebar

Reply:
red herring, #652 

so what does `toc.follow` do?

Reply:
it makes sure the TOC follows the currently visible section of the document, e.g. if you scroll down the page, the TOC on the right will follow where you are